h/static/scripts/login-forms/components/FacebookIcon.tsx

```diff
@@ -0,0 +1,46 @@
+import type { JSX } from 'preact';
+
+export default function FacebookIcon({
```

Comment:
This is a straight copy of the `facebook.svg` icon. It looks to me like it might be possible to simplify it. I'll try running it through SVGO and see what it produces.